subworkflows/nf-scil/preproc_dwi/meta.yml

```
@@ -0,0 +1,113 @@
+name: "preproc_dwi"
+description: |
+  Subworkflow to perform pre-processing of DWI image.
+  It requires at least two input channels, one with DWI image (ch_dwi) and one with b-value and b-vector files (ch_bval_bvec).
+  The third is optional and includes a reversed phase encoded b = 0 image (ch_rev_b0).
+  If a reversed phase encoded b = 0 image (ch_rev_b0) is provided, Topup will run, otherwise the pipelines will
+  run only Eddy correction.
+    1) Required channels include the following main steps:
+          Brain Extraction, Denoising, Eddy, N4 Bias Correction, Normalization, and Resampling.
+    2) Required and optional channels include the following main steps:
+          Brain Extraction, Denoising, Topup+Eddy, N4 Bias Correction, Normalization and Resampling.
+  The resulting file from this subworkflows is a DWI that has been corrected, normalized, cropped and resampled.
+  The next steps would be to combine the resulting DWI-corrected image with reconstruction modules
+  such as RECONST_FODF and/or RECONST_DTIMETRICS.
```

```suggestion
  The next step would be to extract diffusion profiles for the corrected DWI, using a module implementing either DTI (see RECONST_DTI), fODF (see RECONST_FODF) or MAP-MRI (NotImplementedYet).
```

Not required to keep the last, but I like the little easter egg for an eventual user-dev 😛